kubeflow/example/readme.md

```diff
@@ -16,6 +26,8 @@ The dataset contains 7 columns and 330 rows generated by a simulator:
 * Five columns corresponds to measurements from vehicle sensors (mean_acceleration, mean_fuel, mean_speed, duration, route_length) that will be used to categorize the driving profile.
 * Last column corresponds to the target, the driving profile associated to a group of measurements, from 0 to 3.
 
+
```

Comment:
Suggestion: better remove blank lines if not needed

Reply:
Done! LGFM